app/controllers/world_locations_controller.rb

```diff
@@ -79,4 +79,8 @@ def parent_taxon
       description: "Accessing UK services from #{@world_location.name}, advice for travelling to the UK, and help with trading between the UK and #{@world_location.name}."
     }
   end
+
+  def b_variant_page_content
```

Comment:
Don't think we need this anymore given https://github.com/alphagov/whitehall/blob/master/app/controllers/world_locations_controller.rb#L43